designer/client/page-edit.js

```diff
@@ -61,6 +64,7 @@ export class PageEdit extends React.Component {
       ? (copyPage.controller = controller)
       : delete copyPage.controller;
 
+    if (isEditingDeclaration) copy.declaration = formData.get("declaration");
```

Comment:
I don't think you need `isEditingCondition` here. I think copy.declaration = formData.get("declaration") is fine, if it's empty or undefined it shouldn't make a difference? (just double check that though!)

Reply:
For some reason it throws a 401 Unauthorised error and the smoke test [fails](https://github.com/XGovFormBuilder/digital-form-builder/runs/5755561473?check_suite_focus=true) when trying to change the page title without the check, but it does default to null or undefined

Reply:
I've defaulted it to empty if it's null or undefined, should be cleaner